### PR TITLE
:memo: docs(readme): add a part in project usage about how to add ssh key and handle it on github (#3)

### LINKED ISSUE #5: update the documentation about setting the SSH key

### PR DESCRIPTION
**Pull request by niyushabaghayi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +72,39 @@ mkdir -p ./dags ./logs ./plugins ./config
...
+   - Navigate to "SSH and GPG keys" or a similar section.
+
+   - Click "New SSH key" or equivalent.
+
+   - Paste your public key into the provided field and give it a meaningful title.
+
+3. **Updating Docker Compose**: To ensure that your SSH key is correctly mounted in the `git-sync` container, modify the relevant line in your `docker-compose.yaml` file as follows:
+
+   ```yaml
+   - ${GIT_SSH_KEY:-~/.ssh/id_<key-type>}:/root/.ssh/id_<key-type>
```
Comment (mostafaghadimi) [inline README.md]: This one should be the file name. (/root/.ssh/<ssh_private_key_file_name>)

File: README.md [hunk 2]
```diff
@@ -70,6 +72,39 @@ mkdir -p ./dags ./logs ./plugins ./config
...
+### Setting Up Your SSH Key
+
+1. **Generate an SSH Key**: If you don't already have an SSH key, you can generate one using the following steps:
+
+   - Open a Terminal: Open your terminal or command prompt.
+
+   - Generate SSH Key: Run the following command to generate a new SSH key:
+
+     ```bash
+     ssh-keygen -t <key-type>
```
Comment (mostafaghadimi) [inline README.md:84]: add example here like:
```bash
ssh-keygen -t <key-type>
# example
# ssh-keygen -t id_ed25519
```

**Review by mostafaghadimi — COMMENTED** [review]
Wonderful. Good job, @niyushabaghayi.

File: README.md (hunk 1 above)
Comment (niyushabaghayi) [reply README.md]: Done :heavy_check_mark:

File: README.md (hunk 2 above)
Comment (niyushabaghayi) [reply README.md:84]: Done :heavy_check_mark:

**Review by mostafaghadimi — APPROVED** [review]
Approved! 🚀 ❤️

**Pull request merged**